video_xblock/video_xblock.py

```diff
@@ -151,7 +195,7 @@ class VideoXBlock(StudioEditableXBlockMixin, XBlock):
         display_name=_('Download Transcript Allowed'),
         help=_(
             "Allow students to download the timed transcript. A link to download the file appears below the video."
-            " By default, the transcript is a .vtt file. If you want to provide the transcript for download"
+            " By default, the transcript is an .srt or .txt file. If you want to provide the transcript for download"
```

Comment:
mmm... Where is the code that returns `str` and `txt` transcripts for download button?